### PR TITLE
Feature/cmake crashpad

### PR DESCRIPTION
**Pull request by vlussenburg**

### CODE REVIEW

**Comment by KrzaQ** [comment]
The changes themselves look good to me. 👍

I would like to not spam the history, though, so at least squash the "update text.yml" commits together.

**Review by KrzaQ — APPROVED** [review]
As said before, squash is necessary.

File: example-app/src/androidTest/java/backtraceio/backtraceio/ExampleInstrumentedTest.java [hunk 1]
```diff
@@ -16,11 +24,46 @@
...
+        onView(withId(R.id.handledException)).perform(click());
+    }
+
+    @Test
+    public void anr() {
+        onView(withId(R.id.anr)).perform(click());
+    }
+
+    // Will break build, obviously.
+    //@Test
```
Comment (BartoszLitwiniuk) [inline example-app/src/androidTest/java/backtraceio/backtraceio/ExampleInstrumentedTest.java:65]: do we need this comment `//@Test`?

File: example-app/src/androidTest/java/backtraceio/backtraceio/ExampleInstrumentedTest.java [hunk 2]
```diff
@@ -16,11 +24,46 @@
...
+    public void nativeCrash() {
```
Comment (BartoszLitwiniuk) [inline example-app/src/androidTest/java/backtraceio/backtraceio/ExampleInstrumentedTest.java:66]: is this method called anywhere?

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -10,41 +10,118 @@ on:
...
+          morgue --token=${{ secrets.BACKTRACE_API_TOKEN }} --endpoint="https://yolo.sp.backtrace.io" list yolo/android-library --select timestamp --filter=timestamp,greater-than,`date +%s%3N -d "15 minutes ago"` || true
+
+#           if [ $(wc -l < crash.csv) -gt 0 ]; then
+#               echo "Success"
+#           else
+#               exit 1
+#           fi
+
+    #        runs-on: macos-latest # necessary for reactivecircus/android-emulator-runner@v2
+    #     - name: Emulator test
```
Comment (BartoszLitwiniuk) [inline .github/workflows/test.yml:118]: do we need it?

File: example-app/src/androidTest/java/backtraceio/backtraceio/ExampleInstrumentedTest.java (hunk 1 above)
Comment (vlussenburg) [reply example-app/src/androidTest/java/backtraceio/backtraceio/ExampleInstrumentedTest.java:65]: Good catch. it’s left there to easily turn on the test. I need to figure out how to run the test (which crashes the app) without crashing the test. Or accept that the test fails but then we need to add a custom annotation that prevent it from running on “normal runs”.

File: example-app/src/androidTest/java/backtraceio/backtraceio/ExampleInstrumentedTest.java (hunk 2 above)
Comment (vlussenburg) [reply example-app/src/androidTest/java/backtraceio/backtraceio/ExampleInstrumentedTest.java:66]: See the above.

File: .github/workflows/test.yml (hunk 3 above)
Comment (vlussenburg) [reply .github/workflows/test.yml:118]: Maybe. Apilevel 16 is not supported on Sauce, which would be nice to test on. So we may have to boot an old school emulator.

**Pull request merged**